framework/org.eclipse.concierge/src/org/eclipse/concierge/Concierge.java

```diff
@@ -4658,7 +4658,11 @@ public Bundle installBundle(final String location, final InputStream in)
 
 			// TODO: check AdminPermission(new bundle, LIFECYCLE)
 
-			return installNewBundle(this, location, in);
+			if(in == null){

```

Comment:
Thanks ... updated ... is there a way to configure Eclipse IDE to auto format like this?